app/src/main/java/com/example/taskyy/ui/screens/ReminderScreen.kt

```diff
@@ -0,0 +1,332 @@
+            }
+        ) {
+            Text(
+                text = "From",
+                modifier = Modifier.padding(end = 40.dp),
+                fontSize = 15.sp
+            )
+            Text(
+                text = "02:33",
+                modifier = Modifier.padding(end = 40.dp),
+                fontSize = 15.sp
+            )
+            Image(
+                painter = painterResource(id = R.drawable.chevron_forward),
+                contentDescription = "proceed forward arrow",
+                modifier = Modifier.padding(end = 40.dp)
+            )
+        }
+        Row(
+            modifier = Modifier.clickable {
+                //navigate to date picker
+            }
+        ) {
+            Text(
+                text = dateString,
+                color = Color.Black,
+                fontSize = 15.sp
+            )
+
+            ChevronForward(topPadding = 0)
+        }
+    }
+}
+
+@Composable
+fun ItemDescription() {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = Modifier.clickable
+        {
+            //navigate to item description page
+        }
+    ) {
+        Text(
+            text = "Reminder Description",
+            modifier = Modifier.padding(start = 20.dp, top = 20.dp),
+            fontSize = 15.sp
+        )
+        ChevronForward(20)
+    }
+}
+
+@Composable
+fun RowDivider(topPadding: Int) {
+    Row(
+        modifier = Modifier.padding(top = topPadding.dp)
+    ) {
+        Box(
+            modifier = Modifier
+                .weight(1f)
+                .height(1.dp) // Adjust the height of the divider as needed
+                .padding(
+                    bottom = 4.dp,
+                    start = 20.dp,
+                    end = 20.dp
+                ) // Adjust the padding to control the distance between the divider and the content
+        ) {
+            HorizontalDivider(color = Color.LightGray) // Divider with black color
+        }
+    }
+}
+
+@Composable
+fun NewItemRow() {
+    Row(
+        modifier =
+        Modifier
+            .padding(top = 100.dp, start = 20.dp)
+            .clickable {
+                //navigate to name screen
+            },
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        WhiteCircleWithBlackBorder()
+        Text(
+            text = "New Reminder",
+            fontSize = 25.sp,
+            fontWeight = FontWeight.Bold,
+            modifier = Modifier.padding(start = 10.dp)
+        )
+        ChevronForward(0)
+    }
+}
+
+@Composable
+fun ChevronForward(topPadding: Int) {
+    Column(
+        horizontalAlignment = Alignment.End,
+        modifier = Modifier
+            .padding(top = topPadding.dp)
+            .fillMaxWidth()
+    ) {
+        Image(
+            painter = painterResource(id = R.drawable.chevron_forward),
+            contentDescription = "proceed forward arrow",
+            modifier = Modifier.padding(end = 15.dp)
+        )
+    }
```

Comment:
Also here, I'd expect that `ChevronForward` is a standalone icon and not a full row which spans the whole width and the icon is at the end. That keeps it less flexible since you force how this icon is used